src/tests/test_utils/common_test_utils/src/ov_plugin_cache.cpp

```diff
@@ -90,12 +90,13 @@ class TestListener : public testing::EmptyTestEventListener {
 }  // namespace
 
 PluginCache& PluginCache::get() {
+    static std::mutex mutex;
+    std::lock_guard<std::mutex> lock(mutex);
     static PluginCache instance;
```

Comment:
Maybe this `instance` should static member of `PluginCache` and then mutex will be not required at all?

Why synchronization is not required in `core` and `reset` function members?